ui/js/models/resultsets_store.js

```diff
@@ -828,7 +827,7 @@ treeherder.factory('ThResultSetStore', [
                                 .map(jobList => getLastModifiedJobTime(jobList))
                                 .filter(x => x);
                             if (lastModifiedTimes.length) {
-                                var lastModifiedTime = _.max(lastModifiedTimes);
+                                var lastModifiedTime = max(lastModifiedTimes);
                                 // subtract 3 seconds to take in account a possible delay
                                 // between the job requests
                                 lastModifiedTime.setSeconds(lastModifiedTime.getSeconds() - 3);
```

Comment:
For instance, we wouldn't be able to do this operation, if we returned `Math.max(...)` earlier. In which case, we would have to do something like `lastModifiedTime -= 3 * 1000 // subtract 3 seconds`. Essentially, updating all the logic everywhere `lastJobUpdate` is used, to use unix timestamps instead of `Date()` objects.

Reply:
Yeah makes sense, ty